automation/jleechanorg_pr_automation/jleechanorg_pr_monitor.py

```
@@ -711,9 +711,7 @@ def discover_open_prs(self, cutoff_hours: int = 24) -> list[dict]:
             self.logger.info("📭 No recent open PRs discovered")
             return []
 
-        # Use timezone-aware minimum datetime for safe comparison with aware datetimes
-        datetime_min_utc = datetime.min.replace(tzinfo=UTC)
-        recent_prs.sort(key=lambda x: x.get("updated_datetime", datetime_min_utc), reverse=True)
+        recent_prs.sort(key=lambda x: x.get("updated_datetime", datetime.min), reverse=True)
```

Ruff (0.14.11)

Use of `datetime.datetime.min` without timezone information

(DTZ901)